src/features/swap/components/SwapForm.tsx

```diff
@@ -119,46 +123,119 @@ export const SwapForm = () => {
           Swap
         </ButtonCustom>
       </Form>
+
+      <Intents intentRefs={snapshot.context.intentRefs} />
     </div>
   )
 }
 
-function renderIntentOutcome(intentOutcome: Context["outcome"]) {
-  if (!intentOutcome) {
-    return null
-  }
+function Intents({
```

Comment:
`Intents` should be a separate component to avoid overloading the SwapForm itself.

Reply:
Yeah, it will most likely be in own file. I left it here because it's still in progress and just reflects a possibility to shows statuses of intents.

Reply:
Yeah, it will be most likely migrated to own file, as soon as design will be ready. Now it is just an example of what we can do with intents.